### PR TITLE
News: wrapping up 2020

### PR DESCRIPTION
**Pull request by neteler**
... further edits are most welcome!

### CODE REVIEW

File: content/news/2020_12_28_wrapping_up_2020.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+title: "GRASS GIS project: wrapping up 2020"
+date: 2020-12-28T21:12:00+02:00
+layout: "news"
+author: GRASS Development Team
+---
+
+**A look back on 2020**
+
+An usual year comes to an end and we nonetheless saw great success in the GRASS GIS project.
```
Comment (petrasovaa) [inline content/news/2020_12_28_wrapping_up_2020.md:10]:
```suggestion
An unusual year comes to an end and we nonetheless saw great success in the GRASS GIS project.
```

File: content/news/2020_12_28_wrapping_up_2020.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+Here our achievements in a nutshell:
+
+- new GUI startup (GSoC student: Linda Kladivova)
+
+<a href="/images/news/new_grass_website2.png">
+  <img src="/images/news/new_grass_website2.png" alt="New GRASS GIS Website" title="New GRASS GIS Website"
+   width="45%" style="float:right;padding-left:10px;padding-top:20px">
+</a>
```
Comment (petrasovaa) [inline content/news/2020_12_28_wrapping_up_2020.md:18]: Looks like the images are switched?

File: content/news/2020_12_28_wrapping_up_2020.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+title: "GRASS GIS project: wrapping up 2020"
+date: 2020-12-28T21:12:00+02:00
+layout: "news"
+author: GRASS Development Team
+---
+
+**A look back on 2020**
+
+An usual year comes to an end and we nonetheless saw great success in the GRASS GIS project.
+Here our achievements in a nutshell:
```
Comment (veroandreo) [inline content/news/2020_12_28_wrapping_up_2020.md:11]:
```suggestion
Here our most relevant achievements in a nutshell:
```

File: content/news/2020_12_28_wrapping_up_2020.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+<a href="/images/news/grassgui8_first_infobar.png">
+  <img src="/images/news/grassgui8_first_infobar.png" alt="New GRASS GIS 8 graphical user interface startup" title="New GRASS GIS 8 graphical user interface startup"
+   width="45%" style="float:right;padding-left:10px;padding-top:20px">
+</a>
+
+- source code management: issues moved to [GitHub](https://github.com/OSGeo/grass/issues)
+- continuous source code analysis with [GitHub Actions](https://github.com/OSGeo/grass/actions/)
+- three virtual meetings with developers and users to discuss further improvements
+
+We welcome everybody interested to [join us](https://grass.osgeo.org/contribute/) - consider
```
Comment (veroandreo) [inline content/news/2020_12_28_wrapping_up_2020.md:35]: The correct expression AFAIU would be *interested in joining us*

File: content/news/2020_12_28_wrapping_up_2020.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+to discuss in our mailing lists, suggest improvements, spread the word, translate message or
```
Comment (veroandreo) [inline content/news/2020_12_28_wrapping_up_2020.md:36]:
```suggestion
to discuss in our mailing lists, suggest improvements, spread the word, translate messages or
```

**Review by veroandreo — COMMENTED** [review]
some minor comments

File: content/news/2020_12_28_wrapping_up_2020.md (hunk 2 above)
Comment (neteler) [reply content/news/2020_12_28_wrapping_up_2020.md:18]: possibly, I am in a hurry :-)

File: content/news/2020_12_28_wrapping_up_2020.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+<a href="/images/news/grassgui8_first_infobar.png">
+  <img src="/images/news/grassgui8_first_infobar.png" alt="New GRASS GIS 8 graphical user interface startup" title="New GRASS GIS 8 graphical user interface startup"
+   width="45%" style="float:right;padding-left:10px;padding-top:20px">
+</a>
+
+- source code management: issues moved to [GitHub](https://github.com/OSGeo/grass/issues)
+- continuous source code analysis with [GitHub Actions](https://github.com/OSGeo/grass/actions/)
+- three virtual meetings with developers and users to discuss further improvements
+
+We welcome everybody interested to [join us](https://grass.osgeo.org/contribute/) - consider
```
Comment (neteler) [inline content/news/2020_12_28_wrapping_up_2020.md:35]:
```suggestion
We welcome everybody interested in [joining us](https://grass.osgeo.org/contribute/) - consider
```

File: content/news/2020_12_28_wrapping_up_2020.md [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+- new releases of [GRASS 7.8.3](https://trac.osgeo.org/grass/wiki/Release/7.8.3-News),
+  [GRASS 7.8.4](https://trac.osgeo.org/grass/wiki/Release/7.8.4-News) followed
+  by [GRASS 7.8.5](https://trac.osgeo.org/grass/wiki/Release/7.8.5-News),
+  altogether with more than ​450 fixes and improvements
+- entirely new website, based on Hugo (co-funded by OSGeo)
+
+<a href="/images/news/grassgui8_first_infobar.png">
+  <img src="/images/news/grassgui8_first_infobar.png" alt="New GRASS GIS 8 graphical user interface startup" title="New GRASS GIS 8 graphical user interface startup"
+   width="45%" style="float:right;padding-left:10px;padding-top:20px">
+</a>
```
Comment (neteler) [inline content/news/2020_12_28_wrapping_up_2020.md:29]:
```suggestion

<a href="/images/news/grassgui8_first_infobar.png">
  <img src="/images/news/grassgui8_first_infobar.png" alt="New GRASS GIS 8 graphical user interface startup" title="New GRASS GIS 8 graphical user interface startup"
   width="45%" style="float:right;padding-left:10px;padding-top:20px">
</a>

- new releases of [GRASS 7.8.3](https://trac.osgeo.org/grass/wiki/Release/7.8.3-News),
  [GRASS 7.8.4](https://trac.osgeo.org/grass/wiki/Release/7.8.4-News) followed
  by [GRASS 7.8.5](https://trac.osgeo.org/grass/wiki/Release/7.8.5-News),
  altogether with more than ​450 fixes and improvements
- entirely new website, based on Hugo (co-funded by OSGeo)

<a href="/images/news/new_grass_website2.png">
  <img src="/images/news/new_grass_website2.png" alt="New GRASS GIS Website" title="New GRASS GIS Website"
   width="45%" style="float:right;padding-left:10px;padding-top:20px">
</a>

```

Comment (neteler) [reply content/news/2020_12_28_wrapping_up_2020.md:29]: align image order to text order

File: content/news/2020_12_28_wrapping_up_2020.md (hunk 2 above)
Comment (neteler) [reply content/news/2020_12_28_wrapping_up_2020.md]: thanks

**Comment by neteler** [comment]
The "get involved" part could still be stronger, ideas welcome

File: content/news/2020_12_28_wrapping_up_2020.md [hunk 8]
```diff
@@ -0,0 +1,43 @@
+---
+title: "GRASS GIS project: wrapping up 2020"
```
Comment (wenzeslaus) [inline content/news/2020_12_28_wrapping_up_2020.md:2]: Title case:

```suggestion
title: "GRASS GIS Project: Wrapping up 2020"
```

**Comment by neteler** [comment]
Thanks for your reviews!

**Pull request merged**